app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+# Actions for sessions
+class SessionsController < ApplicationController
+  def new; end
+
+  def create
+    user = find_user
+    if user&.authenticate(params[:session][:password])
+      log_in user
+      remember user
```

Comment:
`log_in user` is not necessary. `remember user` is sufficient.

Reply:
## Deleted unnecessary method